src/EnergyPlus/PluginManager.cc

```diff
@@ -45,6 +45,12 @@
 // OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <dlfcn.h>
+#endif
```

Comment:
This allows accessing an external shared library's exported symbols at runtime.  On Windows, `_WIN32` will be defined for both 32 or 64 bit builds, and results in importing symbols from `windows.h`.  Otherwise it gets `dlfcn.h`.